src/panel/PanelManager.jsx

```diff
@@ -17,8 +17,8 @@ import { PanelContext } from 'src/libs/panelContext.js';
 import NoResultPanel from 'src/panel/NoResultPanel';
 
 const directionConf = nconf.get().direction;
-
 const directSearchRouteName = 'Direct search query';
+const initialQueryParams = parseQueryString(window.location.search);
```

Comment:
There is no more reason to put that outside `componentDidMount`. You can simply revert this file to its master version.